### ISSUE TITLE
MEMORY ALLOC FAILED: VirtualAlloc faile creeps in more often 

### ISSUE THREAD

**Issue by imrecsoka** [aaronsace/MultiPoolMiner#170]
I have noticed that by the time the benchmarking is done for multipool miner or nicehash miner  then i get the MEMORY ALLOC FAILED: VirtualAlloc failed.  errors and then the results are skewed.
How is it possible to completely always lock the memory or whatever the term is ?
I have allocated my user access  to this i the local policy editor area...  its just that after a few attempts of benchmarks runs then the errors come through almost as if there is a memory leak.

**Comment by aaronsace** [comment]
Is this in PowerShell? Where are you see this error?

**Comment by Jaerin** [comment]
I've noticed this when I had the intensity set too high on certain algorithms, like Blake2s and Lyra2v2.  Setting the intensity down 1-2 below the default seemed to resolve the issues and didn't affect hashrates that much.

**Comment by aaronsace** [comment]
@Jaerin Ah, sgminer. What card are you using and what settings are you using? 

@imrecsoka Edit the sgminer file in notepad from the 'miners' folder.

**Comment by Jaerin** [comment]
Nevermind this is likely a different issue then. What I was seeing was on ccminer.   I was thinking it the illegal memory access problem, but I guess this message is different. My apologies

**Comment by imrecsoka** [comment]
My error - MEMORY ALLOC FAILED: VirtualAlloc failed- Coming from XMR-STAK-CPU.

Since SGminer was mentioned:
Really hate SGminer or CGminer they are just the worst i have used. I dont know why they need to clear the screen. It makes me mad because they just go back to the command prompt with no information whatsoever its a pain to troubleshoot. So people end up asking questions on forums and wait days for ridiculous answers.
Anyways  i hope you guys can help me because im losing it here , really.
I dont understand how i was mining fine and then all of a sudden the mining doesnt start at all.
CGminer and SGminer wont work they just hang
I have tried to re-install drivers and what not, i dont understand what the hell is going on.


It is not there when i start the miner, but after i stop and surf the internet a bit and then strart mining again the memory leaks or errors are there.
I have 8 GB memory windows 7 ultimate:
Windows has checked the file system and found no problems.
  66648063 KB total disk space.
  20 GB free.
I have left my VM like this:
![image](https://user-images.githubusercontent.com/25494601/27264395-a8469e9e-547d-11e7-892f-0dc5d69f7d64.png)

I have tried to add -T --verbose and it give the best information BUT it doesnt still really explain why something is broken.
